org.osgi.test.common/pom.xml

```diff
@@ -31,6 +31,13 @@
 		OSGi Test Common Support
 	</description>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.osgi</groupId>
+			<artifactId>osgi.core</artifactId>
```

Comment:
If our intent is to make proper bundles of these jars, we need to make sure we use the lowest framework version possible to accomplish our goals. So using Core R7 means no Core R6 users can use our bundles.

Reply:
good point, reducing to 6